src/main/java/com/rewedigital/composer/composing/IncludeMarkupHandler.java

```diff
@@ -41,6 +41,10 @@ private ContentRange contentRange() {
         return contentMarkupHandler.assets();
     }
 
+    List<IncludedService> includedServices() {
```

Comment:
I thought about this, however we have three VisibleForTesting on the classpath, and none is from a dependency that is referenced in our pom directly. I would prefer to add my own annotation?

Reply:
I agree and apologize, because I used one of the three in my last PR. Should we add our own dependency to whatever small annotation lib provides this annotation? I don't think we should implement our own annotation

Reply:
Used VisibleForTesting from guava for now.